lib/worker.js

```diff
@@ -1,18 +1,28 @@
 /*global Worker */
 var vms = {};
 
-function runTest(name, type, complete) {
-  function doIt() {
-    vm.onmessage = ({data}) => {
-      vm.onmessage = undefined;
-      complete(data.result || data);
-    };
-    vm.postMessage({name});
-  }
+if (window.Worker != undefined) {
+  function runTest(name, type, complete) {
```

Comment:
Could you move the Worker check inside the `runTest` function so that the function isn't defined twice?